src/OmniSharp/app.config

```diff
@@ -1,56 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Where did this move to? These binding redirects are really important.

Reply:
they didn't move anywhere... I'll bring them back 😱 😨

Reply:
yeah, we'll need em.